### DISCUSSION TITLE
CoCalc and the PreTeXt-Runestone Open Source Ecosystem

### DISCUSSION THREAD

**Discussion by StevenClontz** [sagemathinc/cocalc#6476] [General]
The [PreTeXt-Runestone Open Source Ecosystem](https://prose.runestone.academy/) has received an award scoping the development of a managing organization that will support open-source STEM OER by maintaining PreTeXt and Runestone products and the community surrounding these projects.

For a long time I've felt the CoCalc service was the best way to get folks productive in authoring PreTeXt documents. However, we recently changed our recommended workflow for new users to utilize GitHub Codespaces instead. There are a few reasons for this, which I'm happy to discuss here if there's interest.

I would also like to invite CoCalc to send a representative to our [June 12-16 workshop](https://prose.runestone.academy/workshop) to engage in planning Phase II of this project. Please let me know if you'd like to be involved.

**Comment by williamstein** [comment]
Congrats on your new grant/award!!

 I'm sorry that we at CoCalc weren't able to devote sufficient resources to supporting authoring of PreTeXt documents.
